samples/ClassroomLabs/Scripts/BringImageToSharedImageGallery/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+This script is used to bring a custom image from an Azure virtual machine (VM) to shared image gallery.
+
+## Notes
+- The script must be executed using administrator privileges.
+
+# Directions
+1. Open a PowerShell window.  Make sure that the window notes it is running under administrator privileges.
```

Comment:
Should we tell users that they must have Azure Powershell modules already Installed?  Or they need to login?  I'm not sure if this is assumed...